Bangazon/Controllers/PaymentTypesController.cs

```diff
@@ -0,0 +1,172 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System.Data;
+using System.Data.SqlClient;
+using Dapper;
+using Microsoft.AspNetCore.Http;
+using Bangazon.Models;
+
+namespace Bangazon.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class PaymentTypesController : ControllerBase
+    {
+        private readonly IConfiguration _config;
+
+        public PaymentTypesController(IConfiguration config)
+        {
+            _config = config;
+        }
+
+        public IDbConnection Connection
+        {
+            get
+            {
+                return new SqlConnection(_config.GetConnectionString("DefaultConnection"));
+            }
+        }
+
+
+        // GET api/paymenttypes
+        [HttpGet]
+        public async Task<IActionResult> Get(string q)
```

Comment:
The ticket does mention anything about a `q` parameter. It is not needed.

Reply:
I have made the proper changes to this